packages/labs/compiler/src/lib/template-transform.ts

```diff
@@ -283,6 +283,13 @@ class CompiledTemplatePass {
       'pre:node': (node): boolean | void => {
         if (isElementNode(node)) {
           const attributesToRemove = new Set<unknown>();
+          if (node.tagName.includes(marker)) {
+            // Do not compile template if a marker was inserted in tag name
+            // position.
+            // TODO: Provide a diagnostic here.
```

Comment:
Nit. I think we like to add a username to TODOs so we know who might have context.

Reply:
Agree – done